package.json

```diff
@@ -68,67 +71,70 @@
   },
   "dependencies": {
     "async": "^2.5.0",
-    "autoprefixer": "^8.5.2",
+    "autoprefixer": "^9.5.1",
     "axe-core": "3.0.3",
-    "babel-loader": "^7.1.2",
+    "babel-loader": "^8.0.5",
     "babel-polyfill": "^6.23.0",
-    "browserslist-config-terra": "^1.0.0",
     "chai": "^4.2.0",
     "chalk": "^2.4.2",
-    "clean-webpack-plugin": "^0.1.17",
+    "clean-webpack-plugin": "^2.0.1",
     "clone": "^2.1.1",
     "commander": "^2.19.0",
-    "css-loader": "^0.28.7",
+    "css-loader": "^2.1.1",
+    "cssnano": "^4.1.10",
+    "cssnano-preset-advanced": "^4.0.7",
     "duplicate-package-checker-webpack-plugin": "^3.0.0",
     "express": "^4.16.4",
-    "file-loader": "^1.1.5",
-    "fs-extra": "^6.0.1",
+    "file-loader": "^3.0.1",
+    "fs-extra": "^7.0.1",
```

Comment:
should be able to update this to `^8.0.1`

Reply:
ah, this was release 3 days ago